zulipterminal/ui_tools/boxes.py

```diff
@@ -1361,6 +1361,30 @@ def main_view(self) -> List[Any]:
         ]
         return [part for part, condition in parts if condition]
 
+    def update_msg_content_header(self) -> bool:
+        """
+        Update the content header of a message box (if required).
+        """
+        update_required = False
+        content_header = None
+
+        # Get message content header (if present)
+        if self.need_recipient_header():
+            content_header = self[1][1]
+        elif isinstance(self[0][1], urwid.Text):
+            content_header = self[0][1]
+
+        # Update the content header only if it's present and non-empty.
+        if content_header is not None:
+            update_required = content_header.text != " "
+
+        if update_required:
+            # Re initialize the message if update is required.
+            # FIXME: Render specific element (here content header) instead?
```

Comment:
This doesn't seem too much of an issue performance-wise right now, but let's open an issue for improving this.